prow/plugins/cla/cla_test.go

```diff
@@ -170,3 +170,219 @@ func TestCLALabels(t *testing.T) {
+			context:    "cla/linuxfoundation",
+			state:      "success",
+			issueState: "open",
+			SHA:        "sha",
+			action:     "created",
+			body:       "/check-cla",
+			pullRequests: []github.PullRequest{
+				{Number: 3, Head: github.PullRequestBranch{SHA: "sha"}},
+			},
+
+			addedLabel: fmt.Sprintf("/#3:%s", claYesLabel),
+		},
+		{
+			name:       "cla/linuxfoundation status adds the cla-yes label and removes cla-no label when its state is \"success\"",
+			context:    "cla/linuxfoundation",
+			state:      "success",
+			issueState: "open",
+			SHA:        "sha",
+			action:     "created",
+			body:       "/check-cla",
+			pullRequests: []github.PullRequest{
+				{Number: 3, Head: github.PullRequestBranch{SHA: "sha"}},
+			},
+			hasCLANo: true,
+
+			addedLabel:   fmt.Sprintf("/#3:%s", claYesLabel),
+			removedLabel: fmt.Sprintf("/#3:%s", claNoLabel),
+		},
+		{
+			name:       "cla/linuxfoundation status adds the cla-no label when its state is \"failure\"",
+			context:    "cla/linuxfoundation",
+			state:      "failure",
+			issueState: "open",
+			SHA:        "sha",
+			action:     "created",
+			body:       "/check-cla",
+			pullRequests: []github.PullRequest{
+				{Number: 3, Head: github.PullRequestBranch{SHA: "sha"}},
+			},
+
+			addedLabel: fmt.Sprintf("/#3:%s", claNoLabel),
+		},
+		{
+			name:       "cla/linuxfoundation status adds the cla-no label and removes cla-yes label when its state is \"failure\"",
+			context:    "cla/linuxfoundation",
+			state:      "failure",
+			issueState: "open",
+			SHA:        "sha",
+			action:     "created",
+			body:       "/check-cla",
+			pullRequests: []github.PullRequest{
+				{Number: 3, Head: github.PullRequestBranch{SHA: "sha"}},
+			},
+			hasCLAYes: true,
+
+			addedLabel:   fmt.Sprintf("/#3:%s", claNoLabel),
+			removedLabel: fmt.Sprintf("/#3:%s", claYesLabel),
+		},
+		{
+			name:       "cla/linuxfoundation status retains the cla-yes label and removes cla-no label when its state is \"success\"",
+			context:    "cla/linuxfoundation",
+			state:      "success",
+			issueState: "open",
+			SHA:        "sha",
+			action:     "created",
+			body:       "/check-cla",
+			pullRequests: []github.PullRequest{
+				{Number: 3, Head: github.PullRequestBranch{SHA: "sha"}},
+			},
+			hasCLANo:  true,
+			hasCLAYes: true,
+
+			removedLabel: fmt.Sprintf("/#3:%s", claNoLabel),
+		},
+		{
+			name:       "cla/linuxfoundation status retains the cla-no label and removes cla-yes label when its state is \"failure\"",
+			context:    "cla/linuxfoundation",
+			state:      "failure",
+			issueState: "open",
+			SHA:        "sha",
+			action:     "created",
+			body:       "/check-cla",
+			pullRequests: []github.PullRequest{
+				{Number: 3, Head: github.PullRequestBranch{SHA: "sha"}},
+			},
+			hasCLANo:  true,
+			hasCLAYes: true,
+
+			removedLabel: fmt.Sprintf("/#3:%s", claYesLabel),
+		},
+	}
+	for _, tc := range testcases {
```

Comment:
Consider taking advantage of `t.Run(tc.name, func() { ... })` (I'd like us to start using this pattern, where possible)

https://golang.org/pkg/testing/#T.Run